js/tabs.js

```diff
@@ -0,0 +1,56 @@
+var tabs = document.querySelectorAll('.js-tab');
+
+function setupTabs() {
+  for (var i = 0; i < tabs.length; i += 1) {
+    var tab = tabs[i],
+        tabSelector = tab.getAttribute('href'),
+        tabContent = document.querySelector(tabSelector);
+
+    if (tabContent) {
+      tab.setAttribute('aria-selected', 'false');
+      tabContent.classList.add('u-hide');
+
+      tab.addEventListener('click', function(e) {
+        e.preventDefault();
+        toggleTab(event.currentTarget);
```

Comment:
Shouldn't this be `e.currentTarget`? `event` doesn't appear to be defined anywhere.